app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    file_source = command[1]
+    file_new = command[2]
+    if file_source == file_new:
```

Comment:
You should additionally check that the command is exactly `"cp"` as mentioned [in the checklist](https://github.com/rinotokun/py-copy-file/blob/develop/checklist.md#code-efficiency).